tools/ktx/command.h

```diff
@@ -186,15 +187,28 @@ enum class OutputFormat {
     json_mini,
 };
 
+/**
+//! [command options_format]
+<dl>
+    <dt>--format text | json | mini-json</dt>
+    <dd>Specifies the output format. Possible options are: <br />
```

Comment:
This header file is in fact included by every command.

> if a command's output has some other focus it would not expose this option

Exactly, but "report output format" sounds good to me. Will change it.